src/components/NavBar/NavigationBar.tsx

```diff
@@ -0,0 +1,174 @@
+"use client";
+
+import React from "react";
+import AppBar from "@mui/material/AppBar";
+import IconButton from "@mui/material/IconButton";
+import MenuIcon from "@mui/icons-material/Menu";
+import SwipeableDrawer from "@mui/material/SwipeableDrawer";
+import Link from "next/link";
+import { styled } from "styled-components";
```

Comment:
`import styled`, not `import { styled }`, please :)

Reply:
Done